### ISSUE TITLE
Failover fails with "socket: permission denied"

### ISSUE THREAD

**Issue by tribut** [eduvpn/python-eduvpn-client#533]
# Issue

Network with restricted UDP connectivity does not trigger failover from Wireguard to OpenVPN/TCP. "Downloaded" stays at "0 B" in the GUI.
Manually connecting using `eduvpn-cli connect -t` works fine.

# Installation

`apt install eduvpn-client`

# Version

`4.1.1-1+ubuntu+23.04+1`

# Check if your issue is in the known issues list

Not in list.

# OS/Distribution

Ubuntu 23.04

# Logs

~~~
$ eduvpn-gui -d
2023-06-05 10:02:01,156 - MainThread - DEBUG - eduvpn.ui.app - app.py:70 - activate
2023-06-05 10:02:01,215 - <function ApplicationModelTransitions.get_previous_servers at 0x7f5a402a3060> - DEBUG - eduvpn.app - app.py:37 - Transition: NO_SERVER, old state: DEREGISTERED
2023-06-05 10:02:02,768 - MainThread - DEBUG - eduvpn.ui.ui - ui.py:989 - activated server: <eduvpn_common.server.InstituteServer object at 0x7f5a2b84df50>
2023-06-05 10:02:02,770 - <function ApplicationModelTransitions.loading_server at 0x7f5a402a32e0> - DEBUG - eduvpn.app - app.py:61 - Transition: LOADING_SERVER, old state: NO_SERVER
2023-06-05 10:02:02,771 - <function ApplicationModelTransitions.chosen_server at 0x7f5a402a34c0> - DEBUG - eduvpn.app - app.py:66 - Transition: CHOSEN_SERVER, old state: LOADING_SERVER
2023/06/05 10:02:02 - Go - DEBUG - Getting access token
2023/06/05 10:02:02 - Go - DEBUG - Access token is not expired, returning
2023-06-05 10:02:02,799 - <function ApplicationModelTransitions.authorized at 0x7f5a402a3c40> - DEBUG - eduvpn.app - app.py:86 - Transition: AUTHORIZED, old state: CHOSEN_SERVER
2023-06-05 10:02:02,800 - <function ApplicationModelTransitions.parse_request_config at 0x7f5a402b4040> - DEBUG - eduvpn.app - app.py:97 - Transition: REQUEST_CONFIG, old state: AUTHORIZED
2023/06/05 10:02:02 - Go - DEBUG - Getting access token
2023/06/05 10:02:02 - Go - DEBUG - Access token is not expired, returning
2023/06/05 10:02:02 - Go - DEBUG - Getting access token
2023/06/05 10:02:02 - Go - DEBUG - Access token is not expired, returning
2023-06-05 10:02:03,282 - <function ApplicationModelTransitions.parse_config at 0x7f5a402b4220> - DEBUG - eduvpn.app - app.py:102 - Transition: DISCONNECTED, old state: REQUEST_CONFIG
2023/06/05 10:02:03 - Go - DEBUG - Running token callback
2023-06-05 10:02:03,295 - call-model - DEBUG - eduvpn.app - app.py:340 - Save tokens called
2023-06-05 10:02:03,330 - <function ApplicationModelTransitions.parse_connecting at 0x7f5a402b4720> - DEBUG - eduvpn.app - app.py:125 - Transition: CONNECTING, old state: DISCONNECTED
2023-06-05 10:02:03,330 - call-model - DEBUG - eduvpn.nm - nm.py:465 - writing wireguard configuration to Network Manager
2023-06-05 10:02:03,333 - MainThread - DEBUG - eduvpn.ui.ui - ui.py:1082 - clicked on switch connection state
2023-06-05 10:02:03,333 - call-model - DEBUG - eduvpn.nm - nm.py:421 - setting default gateway: True
2023-06-05 10:02:03,334 - MainThread - DEBUG - eduvpn.nm - nm.py:369 - Adding new connection
2023-06-05 10:02:03,341 - MainThread - DEBUG - eduvpn.nm - nm.py:792 - Connection added for uuid: e600a172-55b6-457b-97e1-e177cdb40db0
2023-06-05 10:02:03,344 - MainThread - DEBUG - eduvpn.nm - nm.py:558 - activate_connection: <NM.RemoteConnection object at 0x7f5a2b872540 (NMRemoteConnection at 0x21a6620)>
2023-06-05 10:02:03,355 - MainThread - DEBUG - eduvpn.nm - nm.py:573 - activate_connection_async result: <NM.ActiveConnection object at 0x7f5a43866e40 (NMActiveConnection at 0x21c69b0)>
2023-06-05 10:02:03,511 - <function ApplicationModelTransitions.parse_connected at 0x7f5a402b4680> - DEBUG - eduvpn.app - app.py:120 - Transition: CONNECTED, old state: CONNECTING
2023-06-05 10:02:03,516 - MainThread - DEBUG - eduvpn.app - app.py:165 - Current protocol is WireGuard, failover should continue
2023-06-05 10:02:03,518 - call-model - DEBUG - eduvpn.app - app.py:206 - starting failover with gateway 10.30.0.1 and MTU 1420 for protocol WireGuard
2023-06-05 10:02:03,518 - call-model - DEBUG - eduvpn.app - app.py:228 - Failed to start failover, error: socket: permission denied
~~~

**Comment by jwijenbergh** [comment]
Hmmm that is so weird. Ubuntu 23.04 changed this then?

You can set it yourself (`sysctl -w net.ipv4.ping_group_range="0 2147483647"`) but maybe not the best idea to start changing values before we understand why they changed it.

**Comment by jwijenbergh** [comment]
This is something we cannot fix, tracking failover UI improvements in #543
